commit 3e9f411275c50bb25ef8108c64d39d9c11ee2f74 (refs/changes/67/2767/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-10-11T01:16:56+00:00 (2 years ago)
    
    MORAY-443 Clean shutdown broken by MORAY-421
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Reviewed by: Jason King <jason.king@joyent.com>
    Reviewed by: Kody A Kantor <kody.kantor@gmail.com>
    Approved by: Kody A Kantor <kody.kantor@gmail.com>

diff --git a/lib/server.js b/lib/server.js
index ff11000..c765bb2 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -246,8 +246,8 @@ MorayServer.prototype.close = function close() {
     barrier.on('drain', function () {
         self.fast_server.close();
 
-        if (self.kang_server !== null) {
-            self.kang_server.close();
+        if (self.monitor_server !== null) {
+            self.monitor_server.close();
         }
 
         self.emit('close');
